apps/rehabstod/src/components/SettingsDialog/SelectCareUnits.tsx

```diff
@@ -0,0 +1,36 @@
+import { IDSSelect } from '@frontend/ids-react-ts'
+import { useId } from 'react'
+import { UserPreferences } from '../../schemas'
+import { useGetUserQuery } from '../../store/api'
+import { getUnitsForUser } from '../../utils/getUnitsForUser'
+
+export function SelectCareUnits({ onChange, preferences }: { onChange: (value: string) => void; preferences: UserPreferences }) {
+  const { isLoading, data: user } = useGetUserQuery()
+  const id = useId()
+  if (!user || isLoading) {
+    return null
+  }
+
+  return (
+    <IDSSelect>
+      <label className="mb-0" htmlFor={id}>
+        Förvald enhet
+      </label>
+      <select
+        id={id}
+        name="options"
+        className="text-neutral-20 my-3 box-border w-full truncate rounded border py-3 px-5 text-left"
```

Comment:
I believe we can remove `my-3` to get rid of that extra margin we noticed in the demo. `IDSSelect` adds the same margin so it probably became doubled.